src/DssSpell.sol

```diff
@@ -58,29 +57,46 @@ contract DssSpellAction is DssAction {
-    address constant MCD_CLIP_ETH_B       = 0x71eb894330e8a4b96b8d6056962e7F116F50e06F;
-    address constant MCD_CLIP_CALC_ETH_B  = 0x19E26067c4a69B9534adf97ED8f986c49179dE18;
-    address constant MCD_CLIP_ETH_C       = 0xc2b12567523e3f3CBd9931492b91fe65b240bc47;
-    address constant MCD_CLIP_CALC_ETH_C  = 0x1c4fC274D12b2e1BBDF97795193D3148fCDa6108;
-    address constant MCD_CLIP_WBTC_A      = 0x0227b54AdbFAEec5f1eD1dFa11f54dcff9076e2C;
-    address constant MCD_CLIP_CALC_WBTC_A = 0x5f4CEa97ca1030C6Bd38429c8a0De7Cd4981C70A;
+    address constant MCD_CLIP_BAT_A             = 0x3D22e6f643e2F4c563fD9db22b229Cbb0Cd570fb;
+    address constant MCD_CLIP_CALC_BAT_A        = 0x2e118153D304a0d9C5838D5FCb70CEfCbEc81DC2;
+    address constant MCD_CLIP_ZRX_A             = 0xdc90d461E148552387f3aB3EBEE0Bdc58Aa16375;
+    address constant MCD_CLIP_CALC_ZRX_A        = 0xebe5e9D77b9DBBA8907A197f4c2aB00A81fb0C4e;
+    address constant MCD_CLIP_KNC_A             = 0x006Aa3eB5E666D8E006aa647D4afAB212555Ddea;
+    address constant MCD_CLIP_CALC_KNC_A        = 0x82c41e2ADE28C066a5D3A1E3f5B444a4075C1584;
+    address constant MCD_CLIP_MANA_A            = 0xF5C8176E1eB0915359E46DEd16E52C071Bb435c0;
+    address constant MCD_CLIP_CALC_MANA_A       = 0xABbCd14FeDbb2D39038327055D9e615e178Fd64D;
+    address constant MCD_CLIP_COMP_A            = 0x2Bb690931407DCA7ecE84753EA931ffd304f0F38;
+    address constant MCD_CLIP_CALC_COMP_A       = 0x1f546560EAa70985d962f1562B65D4B182341a63;
+    address constant MCD_CLIP_LRC_A             = 0x81C5CDf4817DBf75C7F08B8A1cdaB05c9B3f70F7;
+    address constant MCD_CLIP_CALC_LRC_A        = 0x6856CCA4c881CAf29B6563bA046C7Bb73121fb9d;
+    address constant MCD_CLIP_BAL_A             = 0x6AAc067bb903E633A422dE7BE9355E62B3CE0378;
+    address constant MCD_CLIP_CALC_BAL_A        = 0x79564a41508DA86721eDaDac07A590b5A51B2c01;
+    address constant MCD_CLIP_UNI_A             = 0x3713F83Ee6D138Ce191294C131148176015bC29a;
+    address constant MCD_CLIP_CALC_UNI_A        = 0xeA7FE6610e6708E2AFFA202948cA19ace3F580AE;
+    address constant MCD_CLIP_RENBTC_A          = 0x834719BEa8da68c46484E001143bDDe29370a6A3;
+    address constant MCD_CLIP_CALC_RENBTC_A     = 0xcC89F368aad8D424d3e759c1525065e56019a0F4;
+    address constant MCD_CLIP_AAVE_A            = 0x8723b74F598DE2ea49747de5896f9034CC09349e;
+    address constant MCD_CLIP_CALC_AAVE_A       = 0x76024a8EfFCFE270e089964a562Ece6ea5f3a14C;
 
-    address constant ETHBTC               = 0x81A679f98b63B3dDf2F17CB5619f4d6775b3c5ED;
+    address constant GOV_MULSTISIG  = 0x73f09254a81e1F835Ee442d1b3262c1f1d7A13ff;
+    address constant RISK_MULSTISIG = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
+    address constant RWF_MULSTISIG  = 0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09;
+    address constant GRO_MULSTISIG  = 0x7800C137A645c07132886539217ce192b9F0528e;
+    address constant CP_MULSTISIG   = 0x6A0Ce7dBb43Fe537E3Fd0Be12dc1882393895237;
```

Comment:
same, this is not a mutisig.

Reply:
This matches [MIP40c3-SP5](https://forum.makerdao.com/t/mip40c3-sp5-core-unit-budget-mkt-001/6824)